src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -220,10 +220,7 @@ public DynamoPreferencesData DynamoPreferences
         /// of Graph.Json.
         /// </summary>
         [JsonProperty("Camera")]
-        public CameraData Camera
-        {
-            get { return DynamoViewModel.BackgroundPreviewViewModel.GetCameraInformation(); }
-        }
+        public CameraData Camera => DynamoViewModel.BackgroundPreviewViewModel.GetCameraInformation() ?? new CameraData();
```

Comment:
`CompareWorkspaceViews` has the camera comparison. I think, running this in self serve CI and if the test is not not broken, then we are good there. Thoughts?

Reply:
hmm.. If we don't have a similar test, can we add a unit test which removes camera before serialization and read the json file to confirm the property is there?

Reply:
I did not get it..removing camera from the file?

Reply:
was the test failing?

Reply:
not locally. Yet to check in self serve CI

Reply:
I guess my question is - shouldn't the test have been failing before and now passing?

Reply:
why would it fail locally? I have the preview.

Reply:
ah ha! okay, thanks, got it.

Reply:
okay, well, if the self serve is really a good test then it also should have failed there though. I think the self serve machine may actually have a GPU... we need to make sure it passes when running the json tests perhaps because there is no GPU on that machine I believe.

Reply:
correct. `a.camera == b.camera` which case both are null earlier.